packages/react-animation/src/useDraggable/useDraggable.ts

```diff
@@ -0,0 +1,94 @@
+import { useMount, useRefValue, useUnmount } from '@mediamonks/react-hooks';
+import gsap from 'gsap';
+import InertiaPlugin from 'gsap/InertiaPlugin';
+import { useRef, type MutableRefObject, type RefObject } from 'react';
+
+/**
+ * When running on a node environment the `window` is not available. Registering the Draggable
+ * plugin will throw errors, therefore we create this little proxy method to import draggable
+ * dynamically,
+ */
+export const getDraggable = async (): Promise<typeof Draggable | undefined> => {
+  if (typeof window === 'undefined') {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const { Draggable } = await import('gsap/Draggable');
+
+  gsap.registerPlugin(Draggable, InertiaPlugin);
+
+  return Draggable;
+};
+
+/**
+ * Omitting all callback properties, use event listener instead.
```

Comment:
The `Omit` below is commented out, is this still valid?

Reply:
From what I could see, this is not valid anymore as we are using these callbacks in useCarousel